packages/editor/src/components/browser/Browser.tsx

```diff
@@ -9,6 +9,7 @@ import { useFuncBrowser } from './function/FunctionBrowser';
 import { useTypeBrowser } from './type/TypeBrowser';
 import { useTableColBrowser } from './tableCol/TableColBrowser';
 import BrowserBody from './BrowserBody';
+import { useRoleSelectorBrowser, type RoleSelectorOptions } from './roleSelector/RoleSelectorBrowser';
```

Comment:
why selector? isn't this the meaning of a browser, to select something?
I would just call ist useRoleBrowser and RoleBrowserOptions or just RoleOptions...